backend/twitterbot/top_contributors.py

```diff
@@ -0,0 +1,40 @@
+import collections
+from datetime import datetime, timedelta
+
+from tournesol.models.comparisons import Comparison
+from tournesol.models.poll import DEFAULT_POLL_NAME
+from tournesol.models.ratings import ContributorRating
+
+
+def get_previous_month_top_public_contributor():
+    """Return the top public contributor of the previous month."""
```

Comment:
Same here, we can use the plural:

```suggestion
    """
    Return the top contributors of the previous month, based on the number of
    their public comparisons.
    """
```